gpkit/varkey.py

```diff
@@ -24,6 +24,8 @@ class VarKey(object):
     VarKey with the given name and descr.
     """
     new_unnamed_id = Counter()
+    subscripts = ["model", "idx"]
+    eq_ignores = frozenset(["units", "value"])
```

Comment:
I know they'll throw an error elsewhere if they don't actually link. I think units just weren't considered because Pints equality is annoying, floating-pointsy, but now that unitstr are compared it's basically like units are compared.... :p